app/mm-redux/client/client4.ts

```diff
@@ -448,6 +448,15 @@ export default class Client4 {
         );
     }
 
+    getKnownUsers = async () => {
+        this.trackEvent('api', 'api_get_known_users');
+
+        return this.doFetch(
+            `${this.getUsersRoute()}/known`,
```

Comment:
what server version was this introduced? are we handling this not to be called on servers that do not support it?

Reply:
This is introduced in 5.23, so yes, we should check it to verify we only clean up the users when we are over that version.

Reply:
No I just check if the version is 5.23 before you try even to remove the users, this fix the existing crash, and leave the "not so bad" behavior of showing users that are no longer visible until that information is refreshed.

Reply:
can you point me to where is this being called ?